docs/pages/server-access/guides/openssh.mdx

```diff
@@ -23,13 +29,36 @@ We've outlined these reasons in [OpenSSH vs Teleport SSH for Servers?](https://g
 
 (!docs/pages/includes/edition-prereqs-tabs.mdx!)
 
-- A Linux host with the OpenSSH server `sshd` installed, but not Teleport. The
-  SSH port on this host must be open to traffic from the Teleport Proxy Service
-  host.
+- A Linux host with the OpenSSH server `sshd` version 7.4 or above installed,
+  but not Teleport. The SSH port on this host must be open to traffic from the
+  Teleport Proxy Service host.
 
 (!docs/pages/includes/tctl.mdx!)
 
-## Step 1/4. Configure `sshd` to trust the Teleport CA
+## Step 1/5. Add a node resource to your Teleport cluster
+
```

Comment:
> Teleport needs to know how to reach your OpenSSH server

I think it would help to clarify the sentence to add a short paragraph at the top of this section indicating what is happening architecturally in the setup documented here. What is the Teleport cluster doing with the `node` resource?

Reply:
Good idea, done